### PR TITLE
Remove calls to column command

### PR DESCRIPTION
**Pull request by mariusSincovici**
* replace `column` command because it is not always available in the system and for snap it cannot be accessed. 
* try to first get the CLI windows size from stdout fd and if it fails try stty

### CODE REVIEW

File: cli/terminal.go [hunk 1]
```diff
@@ -118,3 +123,100 @@ func ReadPlanFromTerminal() (int, error) {
 	}
 	return planID, nil
 }
+
+// Columns formats a list of strings to a tidy column representation
+func Columns(data []string) (string, error) {
```
Comment (piotrjurkiewicz) [inline cli/terminal.go:128]: Is there a need to be public?

File: cli/terminal.go [hunk 2]
```diff
@@ -118,3 +123,100 @@ func ReadPlanFromTerminal() (int, error) {
...
+	}
+	if len(row) > 0 {
+		t.AppendRow(row)
+	}
+
+	return t.Render(), nil
+}
+
+// Gets the size of CLI window
+func CliDimensions() (int, int, error) {
```
Comment (piotrjurkiewicz) [inline cli/terminal.go:199]: Is there a need to be public?

File: go.mod [hunk 3]
```diff
@@ -24,6 +24,7 @@ require (
 	github.com/hako/durafmt v0.0.0-20210608085754-5c1018a4e16b
 	github.com/jbowtie/gokogiri v0.0.0-20190301021639-37f655d3078f
 	github.com/jbowtie/ratago v0.0.0-20200401224626-3140c0a9b186
+	github.com/jedib0t/go-pretty/v6 v6.5.9
```
Comment (piotrjurkiewicz) [inline go.mod:27]: I think that it should be assessed whether we want to add a new external dependency. How much it increases binary sizes? Do we really need it? And if so, whether we can make use of it in other places.

File: cli/terminal.go (hunk 1 above)
Comment (mariusSincovici) [reply cli/terminal.go]: copied like this from the internal package. Changed because it seams to work.

File: go.mod (hunk 3 above)
Comment (mariusSincovici) [reply go.mod]: removed the dependency and used Println instead

**Review by bartoszWojciechO — APPROVED** [review]

**Review by piotrjurkiewicz — APPROVED** [review]

File: cli/terminal.go [hunk 4]
```diff
@@ -118,3 +122,90 @@ func ReadPlanFromTerminal() (int, error) {
 	}
 	return planID, nil
 }
+
+// formats a list of strings to a tidy column representation
+func columns(data []string) (string, error) {
+	width, _, err := cliDimensions()
+	if err != nil {
+		// workaround for tests because terminal size cannot be calculated
```
Comment (devzbysiu) [inline cli/terminal.go:130]: Hmm :thinking: , wouldn't be better to use strategy pattern and inject different strategy based on if we are in the test or not? So for tests we would inject strategy doing `return strings.Join(data, " "), err` and for regular use we would inject `return formatTable(data, width)`. 
Then the strategy could be also tested separately. What do you think?

**Review by devzbysiu — APPROVED** [review]

**Pull request merged**